src/main/java/cms/gov/madie/terminology/controller/VsacController.java

```diff
@@ -57,9 +54,12 @@ public ResponseEntity<String> getValueSet(
   }
 
   protected String serializeFhirValueset(ValueSet fhirValueSet) {
+    return parser.encodeResourceToString(fhirValueSet);
+  }
 
-    String serialized = parser.encodeResourceToString(fhirValueSet);
-
-    return serialized;
+  @PutMapping(path = "/validateCodes", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
ideally, REST endpoints would be lowercase, hyphenated when multiple words, and avoid verbs. Think of them as resources. For example, /validateCodes could be PUT /validations/codes

Reply:
yes make sense, I will update it